snuba/settings/__init__.py

```diff
@@ -40,6 +40,8 @@
 # Storage names to apply dataset partitioning to
 PARTITIONED_STORAGES: Set[str] = set()
 
+PARTITIONED_CLUSTERS: Sequence[Mapping[str, Any]] = []
```

Comment:
Please provide an example of how to use in comments above this line.

Reply:
Also, can you please clarify (in this file ideally) how CLUSTERS and PARTITIONED_CLUSTERS interacts

Reply:
It's a nitpick i know but can you put PARTITIONED_CLUSTERS below CLUSTERS in this file